index.js

```diff
@@ -7,38 +7,36 @@ var app = express();
 var gitbookMiddleware = GitBookMiddleware();
 
 var EVENTS = {
-    'publish': function(payload) {
+    publish: function(payload) {
         var book = payload.book;
         var build = payload.build;
-        return '<'+book.author.urls.profile+'|'+book.author.name+'> published a new update of <'+book.urls.access+'|'+book.title+'> on branch '+build.branch;
+        return `<${book.author.urls.profile}|${book.author.name}> published a new update of <${book.urls.access}|${book.title}> on branch ${build.branch}`;
     },
-    'thread': function(payload) {
+    thread: function(payload) {
         var book = payload.book;
         var thread = payload.thread;
-        return 'Discussion <'+thread.urls.details+'|#'+thread.number+'> "'+thread.title+'" has been ' + payload.action;
+        return `Discussion <${thread.urls.details}|#${thread.number}> "${thread.title}" has been ${payload.action}`;
     },
-    'thread_comment': function(payload) {
+    thread_comment: function(payload) {
         var book = payload.book;
         var thread = payload.thread;
         var comment = payload.comment;
-        return '<'+comment.user.urls.profile+'|'+comment.user.name+'>" commented on discussion <'+thread.urls.details+'|#'+thread.number+'> "'+thread.title+'"';
+        return `<${comment.user.urls.profile}|${comment.user.name}> commented on discussion <${thread.urls.details}|#${thread.number}> "${thread.title}"`;
     }
 };
 
-
 app.get('/', function (req, res) {
     res.redirect('https://github.com/GitbookIO/services-slack');
 });
 
 app.post('/hook/v1/:tokens([\\/.\\w]*|)', gitbookMiddleware, function (req, res, next) {
     var messageBuilder = EVENTS[req.headers['x-gitbook-event']];
-
     if (!messageBuilder) {
         return res.status(200).end();
     }
 
     // Extract payload
-    var payload = req.body.payload;
+    var payload = req.body;
```

Comment:
@yangshun Yep, `.payload` is an attribute of JSON delivered by the webhook, which looks like:

```json
{
    "eventName": "...",
    "payload": {
        ...
        "book": "org/book_id",
        ...
    }
}
```